test/math/ExpDecayMath.ts

```diff
@@ -33,7 +33,7 @@ describe('ExpDecayMath', () => {
                 });
             } else {
                 await expect(expDecayMath.calcExpDecay(ethAmount, timeElapsed, halfLife)).to.revertedWithError(
-                    'Overflow'
+                    'panic code 0x11'
                 );
             }
         });
```

Comment:
A few more tests missing below (not aligned with latest changes in the ExpDecay PR)

Reply:
merged with the latest changes